extension/scripts/features/searchchat/ttSearchChat.js

```diff
@@ -29,8 +29,8 @@
 			const input = parent.find(".tt-chat-filter input");
 			if (!input) return;
 
-			const keyword = input.value;
-			if (keyword) searchChat(message, keyword);
+			const inputValue = input.value;

```

Comment:
Why would you change from a good variable name to a mediocre one?

Reply:
The name ``keyword`` doesn't actually imply its use.
Whereas the name ``inputValue`` says that it is the value of the input.

Reply:
It's the keyword of the search, so it implies its use very well?

Reply:
If it was new code, I wouldn't mind. But it's already a good name, and imo this only makes it worse. 

Sure, it's the value of the input, but what does that really tell me? It's the keyword that the user want to search on.

Reply:
I agree.